temporalio/activity.py

```diff
@@ -238,13 +240,30 @@ def wait_sync(self, timeout: Optional[float] = None) -> None:
         self.thread_event.wait(timeout)
 
 
+def client() -> Client:
+    """Return a Temporal Client for use in the current activity.
+
+    Returns:
+        :py:class:`temporalio.client.Client` for use in the current activity.
+
+    Raises:
+        RuntimeError: When not in an activity.
```

Comment:
Pedantic/non-blocking:

```suggestion
        RuntimeError: When not in an activity or in a testing environment where client wasn't set.
```

Reply:
👍